src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateDivorceApplication.java

```diff
@@ -47,13 +51,18 @@ public CaseDetails<CaseData, State> apply(final CaseDetails<CaseData, State> cas
 
         final Map<String, Object> templateContent;
         final String templateId;
+        LanguagePreference languagePreference = ENGLISH;
 
         if (caseData.getApplicationType().isSole()) {
             templateContent = divorceApplicationSoleTemplateContent.apply(caseData, caseId);
             templateId = DIVORCE_APPLICATION_SOLE;
         } else {
             templateContent = divorceApplicationJointTemplateContent.apply(caseData, caseId);
             templateId = DIVORCE_APPLICATION_JOINT;
+            if (YES.equals(caseData.getApplicant1().getLanguagePreferenceWelsh())
+                && YES.equals(caseData.getApplicant2().getLanguagePreferenceWelsh())) {
+                languagePreference = WELSH;
+            }
```

Comment:
are we sure on this ? it will depend upon whom you are sending notification and consider that applicant's lang preference ?

Reply:
Till now the divorce doc is always generated based on app1 lang pref. 

And this change is based on the requirement mentioned in the ticket

>If both joint applicant selects 'Welsh' as preferred language then welsh application summary should be generated
If only one joint applicant selects 'Welsh' as preferred language then english application summary should be generated